File: 프로그래머스/0/120811. 중앙값 구하기/중앙값 구하기.java
class Solution {
   public int solution(int[] array) {
        int answer = 0;
        // 버블 정렬
        int n = array.length;
        for (int i = 0; i < n - 1; i++) {
            for (int j = 0; j < n - i - 1; j++) {
                if (array[j] > array[j + 1]) {
                    // 두 원소의 위치 교환
                    int temp = array[j];
                    array[j] = array[j + 1];
                    array[j + 1] = temp;
                }
            }
        }

        for(int i = 0; i <= array.length; i++){
            
            // 중앙값 저장
            if ( i == array.length/2){
                answer = array[i];
            }
        }
        return answer;
    }
}

File: 프로그래머스/0/120811. 중앙값 구하기/중앙값 구하기.py
def solution(array):
    # 버블 정렬
    n = len(array)
    for i in range(n - 1):
        for j in range(n - i - 1):
            if array[j] > array[j + 1]:
                # 두 원소의 위치 교환
                array[j], array[j + 1] = array[j + 1], array[j]

    # 중앙값 저장
    answer = array[len(array) // 2]
    return answer